src/components/Chart/DoughnutChart/style.ts

```diff
@@ -38,6 +38,6 @@ export const SourceName = styled.div`
   color: ${COLORS.black};
 `;
 export const Precentage = styled.div`
-  color: ${COLORS_DOUGHNUT.gray};
+  color: ${COLORS_DOUGHNUT.grey};
   letter-spacing: 0.3px;
```

Comment:
Whenever we are styling font related properties we should try to avoid using pixels and use dynamic units instead, pixel get resized in a weird way and are less responsive - especially in font related properties.
[Reference](https://medium.com/@julienetienne/pixels-are-dead-faa87cd8c8b9)
[Reference](https://www.sitepoint.com/understanding-and-using-rem-units-in-css/)